doc/rados/operations/control.rst

```diff
@@ -329,7 +329,7 @@ The ``quorum`` list at the end lists monitor nodes that are part of the current
 
 This is also available more directly::
 
-	$ ./ceph quorum_status
+	ceph quorum_status
```

Comment:
could you keep the `$`? this is an example of command usage with its output. the idea is to show the interaction of input and output. so `$` helps to differentiate the command line from its output.